CHANGES.rst

```diff
@@ -13,6 +13,11 @@ datamodels
 
 - Updated the list of allowed NIRCam CORONMSK values in model schemas. [#4234]
 
+flat_field
+----------
+ - Updated handling of FGS Guider data, which has not been run through ramp
```

Comment:
A minor nit, but it would be useful to mention (briefly) what the update was. E.g. "Updated handling of error arrays for FGS Guider data, which has not ..."

Reply:
Good point, done.